src/lib/loaders/api/index.ts

```diff
@@ -44,6 +44,19 @@ export interface DataLoaderKey {
 export default (opts) => ({
   // Unauthenticated loaders
 
+  /**
+   * The Convection loaders produced by this factory _will_ cache all responses to memcache.
+   */
+  convectionLoaderWithoutAuthenticationFactory: apiLoaderWithoutAuthenticationFactory(
+    convection,
+    "convection",
+    {
+      requestIDs: opts.requestIDs,
+      userAgent: opts.userAgent,
+      appToken: opts.appToken,
```

Comment:
related to the comment above, I think we're ok with not injecting this (i believe it's the `X-Xapp-Token` header of the request made to MP, if one is set)